src/plugins/inputs/otr.js

```diff
@@ -0,0 +1,16 @@
+module.exports = function(network, chan, cmd, args) {
+	if (cmd !== "otr") {
+		return;
+	}
+	if (args.length === 0 || args[0] === "") {
```

Comment:
Well, your example alone does not point out that it's not needed. However, the `shift()` call a few lines below seems to be the reason why we might not need to check for an empty string. The first 2 following examples show why you should be careful of empty strings, while the next 2 show that `shift()` ~~is a side-effect-based piece of turd~~ happens to make it useless:

```js
// Example 1
> "init".trim().split(" ");
[ 'init' ]

// Example 2
> "".trim().split(" ");
[ '' ]

// Example 3
> text = "/otr init".trim();
'/otr init'
> args = text.split(" ");
[ '/otr', 'init' ]
> args.shift().replace("/", "").toLowerCase();
'otr'
> args
[ 'init' ]

// Example 4
> text = "/otr".trim();
'/otr'
> args = text.split(" ");
[ '/otr' ]
> args.shift().replace("/", "").toLowerCase();
'otr'
> args
[]
```

Don't you think we should leave it there just to be safe, until someone courageous edits this to make sure there is never an empty string there? (and doubles this courage with some `shift()` butchering)